vega_sim/scenario/ideal_market_maker/scenario.py

```diff
@@ -133,16 +165,37 @@ def set_up_background_market(
             wallet_pass=AUCTION2_WALLET.passphrase,
             side="SIDE_SELL",
             initial_price=self.initial_price,
+            market_name=market_name,
+            asset_name=asset_name,
+            initial_asset_mint=self.initial_asset_mint,
             tag=str(tag),
         )
 
+        liquidityprovider = OptimalLiquidityProvider(
+            wallet_name=LIQUIDITY.name,
+            wallet_pass=LIQUIDITY.passphrase,
+            num_steps=self.num_steps,
+            market_order_arrival_rate=self.lambda_val,
+            pegged_order_fill_rate=self.kappa,
+            inventory_upper_boundary=self.q_upper,
+            inventory_lower_boundary=self.q_lower,
+            terminal_penalty_parameter=self.alpha,
+            running_penalty_parameter=self.phi,
+            initial_asset_mint=self.initial_asset_mint,
+            market_name=market_name,
+            asset_name=asset_name,
+            entry_step=5,
+            commitamount=self.lp_commitamount,
+        )
+
         env = MarketEnvironment(
             base_agents=[
                 market_maker,
                 tradingbot,
                 randomtrader,
                 auctionpass1,
                 auctionpass2,
+                # liquidityprovider,
```

Comment:
Deliberate?

Reply:
Yep, one can easily add a liquidity provider in the sim just cancel the comment of this line. Don't you like it? If not, I can remove.

Reply:
No it's fine just checking you didn't mean to have it in 😄